code/__DEFINES/speech_channels.dm

```diff
@@ -0,0 +1,5 @@
+// Used to direct channels to speak into.
+#define SAY_CHAN "Say"
+#define RADIO_CHAN "Radio"
+#define ME_CHAN "Me"
+#define OOC_CHAN "OOC"
```

Comment:
Just out of curiosity, why not make them `X_CHANNEL`? Are those three letter gained really making much of a difference to the decrease in readability, as little as it might arguably be?